src/Plaid/Management/PlaidErrorType.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
File should be removed in favor of existing `ErrorType` enum

Reply:
forgot I had those old ones still in there... will remove